tasks/main.yml

```diff
@@ -8,6 +8,9 @@
   when:
     - ha_cluster_enable_repos
 
+- name: Check and prepare role variables
+  include_tasks: shell_{{ ha_cluster_pacemaker_shell }}/check-and-prepare-role-variables.yml  # yamllint disable-line rule:line-length
+
```

Comment:
Revert this change, please. `shell_pcs/check-and-prepare-role-variables.yml` requires pcs to be installed. Hence the order: enable / check repositories, install essential packages (i.e. pacemaker shell), check and prepare variables.

Reply:
@tomjelinek all ha_cluster packages (including qdevice setup) are coming from HA Extension so we cannot even run Install Essentials before check-and-prepare-role-variables.yml.

One option would be to empty out __ha_cluster_role_essential_packages and move all of them to __ha_cluster_fullstack_node_packages instead?

Or sneak this HAE detection into previous step enable-package-repositories.yml snce it is not used for Suse.

Reply:
> Or sneak this HAE detection into previous step `enable-package-repositories.yml` since it is not used for Suse.

Yes, please. That's where this belongs, that's the purpose of that step.

Reply:
@tomjelinek Thank you, i have reverted that ordering and moved it to repo steps.